### DISCUSSION TITLE
There is only queued and complete status of a workflow run?

### DISCUSSION THREAD

**Discussion by dhutsj** [actions/actions-runner-controller#3324] [Questions]
I checked the logs of `actions-runner-controller`, `github-webhook-server` and `runner` logs.

There is only `queued` and `completed` status for a workflow run status.

How about these status described in the API doc https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28
Can be one of: completed, action_required, cancelled, failure, neutral, skipped, stale, success, timed_out, in_progress, queued, requested, waiting, pending

**Comment by ozankasikci** [comment]
Yeah, unfortunately, Github API Documentation is not super accurate. Apparently, there are two fields being used for defining the state in combination, `status` and `conclusion`.

I also observed two possible values for the status field. The rest of the states are defined in the conclusion field.
